hw2/03-count.html

```diff
@@ -6,28 +6,42 @@
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="03-count.js"></script>
   </head>
-  <body>
-    <main>
-      <h1>Exercise 03 - Count</h1>
-      <hr />
-      <div>
-        <label>Highlight all occurences of a word:</label>
-        <input />
-      </div>
-      <hr />
-      <div>
-        It was the best of times, it was the worst of times, it was the age of
-        wisdom, it was the age of foolishness, it was the epoch of belief, it
-        was the epoch of incredulity, it was the season of Light, it was the
-        season of Darkness, it was the spring of hope, it was the winter of
-        despair, we had everything before us, we had nothing before us, we were
-        all going direct to Heaven, we were all going direct the other way—in
-        short, the period was so far like the present period, that some of its
-        noisiest authorities insisted on its being received, for good or for
-        evil, in the superlative degree of comparison only.
+  <body class="bg-dark">
+    <main class="container my-5 text-left w-50">
+      <div class="card bg-light p-3 my-5 mx-auto">
+        <h1 class="text-center">Exercise 03 - Count</h1>
+        <hr />
+        <div>
+          <label for="userInput">Highlight all occurences of a word:</label>
+          <input type="text" id="userInput" />
+        </div>
+        <hr />
+        <div id="text">
+          It was the best of times, it was the worst of times, it was the age of
+          wisdom, it was the age of foolishness, it was the epoch of belief, it
+          was the epoch of incredulity, it was the season of Light, it was the
+          season of Darkness, it was the spring of hope, it was the winter of
+          despair, we had everything before us, we had nothing before us, we
+          were all going direct to Heaven, we were all going direct the other
+          way—in short, the period was so far like the present period, that some
+          of its noisiest authorities insisted on its being received, for good
+          or for evil, in the superlative degree of comparison only.
+        </div>
+        <div id="highlightedText" style="display: none">
+          It was the best of times, it was the worst of times, it was the age of
+          wisdom, it was the age of foolishness, it was the epoch of belief, it
+          was the epoch of incredulity, it was the season of Light, it was the
+          season of Darkness, it was the spring of hope, it was the winter of
+          despair, we had everything before us, we had nothing before us, we
+          were all going direct to Heaven, we were all going direct the other
+          way—in short, the period was so far like the present period, that some
+          of its noisiest authorities insisted on its being received, for good
+          or for evil, in the superlative degree of comparison only.
+        </div>
```

Comment:
This is not good practice. There should not be two divs with the same exact content. Even if one is not displayed this is still repetitive and adds more html than necessary.